generator/_code.py

```diff
@@ -202,19 +202,25 @@ def __str__(self):
         return '{' + text + '}'
 
     def generate_request(self, key, value):
-        parts = [key + ' = {};', value + ' = ' + value + ' || {};']
-        parts.extend(t.generate_request(key, value) for t in self.types)
+        parts = ['if ({}) {{'.format(value), _indent(1, key + ' = {};')]
```

Comment:
These curlies are warping my mind being for the replacement and for JS, maybe we should use `%s`. A follow-up in any event...